tests/dft-pruning/input.dat

```diff
@@ -27,4 +28,16 @@ SCHEMES={
 for scheme in SCHEMES:
   set DFT_PRUNING_SCHEME $scheme
   energy('pbe/sto-3g')
-  compare_integers(SCHEMES[scheme], int(variable('XC GRID TOTAL POINTS')), 'grid porints for '+scheme) #TEST
+  compare_integers(SCHEMES[scheme], int(variable('XC GRID TOTAL POINTS')), 'grid points for '+scheme) #TEST
+
+# screening based on small weights
+set DFT_PRUNING_SCHEME ROBUST
+screening={
+1e-12:45198,
+1e-14:45429,
+1e-16:45679
+}
```

Comment:
Can you format this a bit more inline with general Python formatting?

Reply:
I ran yapf over some parts